src/style/style.css

```diff
@@ -6968,6 +6973,130 @@ fieldset[disabled] .header-search-input {
 textarea.header-search-input {
   height: auto;
 }
+
+/* ======================== */
+/* MAIN LOCALE CONTAINER    */
+/* ======================== */
+
+.main-locale-container {
+  position: absolute;
+  right: 130px;
+  top: 22px;
+  /* margin: 0 0 0 0; */
+}
+@media (max-width: 768px) {
+  .container {
+    max-width: 100%;
+  }
+}
+@media (min-width: 768px) and (max-width: 992px) {
+  .main-locale-container {
+    right: 110px;
+  }
+}
+@media (max-width: 768px) {
+  .main-locale-container {
+    position: absolute;
+    top: 26px;
+    right: 130px;
+  }
+}
+
+.main-locale-container .dropdown .dropdown__trigger .trigger-content {
+  color: #fff;
+  padding: 6px 8px;
+  border: 1px solid transparent;
+  border-radius: 3px;
+  cursor: pointer;
+}
+
+.main-locale-container .dropdown .dropdown__trigger .trigger-content:hover {
+  border: 1px solid #fff;
+}
+
+.main-locale-container .dropdown.dropdown--active .dropdown__trigger .trigger-content {
+  background: #fff;
+  color: #444;
+  border: 1px solid #fff;
+  border-radius: 3px 3px 0px 0px;
+}
+
+.main-locale-container .dropdown__content {
+  display: none;
+  position: absolute;
```

Comment:
After logging in (both mobile and desktop), the language button overlaps the user's login name:
![image](https://user-images.githubusercontent.com/39800563/43417398-e2666f16-9408-11e8-8f03-59f3b6006e42.png)
![image](https://user-images.githubusercontent.com/39800563/43417407-e890b630-9408-11e8-8d3e-e583148cd33b.png)

Reply:
Fixed